CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 3.2.19 (2024-06-30)
+
+### 3.2.18 (2024-06-16)
+
```

Comment:
`CHANGELOG.md` generates automatically, so to avoid conflicts we shouldn't modify value manually